src/lib/Components/Bidding/Screens/ConfirmBid.tsx

```diff
@@ -164,6 +164,9 @@ export class ConfirmBid extends React.Component<ConfirmBidProps, ConfirmBidState
       ARAuctionID: this.props.sale_artwork.sale.id,
       ARAuctionArtworkID: this.props.sale_artwork.artwork.id,
     })
+    NativeModules.ARNotificationsManager.postNotificationName("ARAuctionArtworkRegistrationUpdatedNotification", {
```

Comment:
As [discussed in Slack](https://artsy.slack.com/archives/C9YNS4X32/p1529534078000289?thread_ts=1529527038.000301&cid=C9YNS4X32), this isn't _always_ required (if the user is already registered and is only bidding) but I don't think it'll be a problem – something to watch out for in mocktions.